### DISCUSSION TITLE
Feature suggestion : Embeds with share url

### DISCUSSION THREAD

**Discussion by ajnart** [umami-software/umami#2372] [Ideas]
## Idea
I'd like to be able to only share a certain portion of the data available on the shared page, I was thinking that adding a way to embed this data would be quite useful (to show on another page in my example) 

## Implementation
I think with NextJS's routing you could reduce the items shown on the share page easily (Like a URL param of `show=countries` for example) 
I'm not sure about restricting only a portion of the data on the main share page, it would probably require some database changes

**Comment by FHU-yezi** [comment]
Does the purpose of your change related to access control? If so, user can just modify params and see all the data.

If not, I think it will be a good improvment which can reduce difficulty understanding all the data.
